### ISSUE TITLE
Improve Regex to avoid false positives

### ISSUE THREAD

**Issue by edoardottt** [edoardottt/cariddi#101]
Context: https://github.com/edoardottt/cariddi/discussions/94#discussioncomment-4162561

**Comment by ocervell** [comment]
the BTC address regex is giving a lot of False positives as well.

**Comment by edoardottt** [comment]
yeah I know about that ... the info group has a lot of false positives.

Unfortunately I'm in really busy weeks and I can't take actions on the open issues for now.
